package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "protosaurus",
+  "version": "1.0.0",
+  "private": true,
+  "workspaces":[
+    "website/*",
+    "packages/*"
+  ],
+  "engines": {
+    "node": "16.13.2",
+    "yarn": "1.22.17"
```

Comment:
I was just about to comment on using a "less strict version", but apparently `engines` field is not strict, TIL! https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines